### PR TITLE
[Event logo addition]

### PR DESCRIPTION
**Pull request by alvo254**
*Please read and take into consideration the following:*
- Title your pull request in this format: `[PLACE-YEAR] Brief description of the proposed change`. For example: `[CHI-2024] Add Bluth Company as a sponsor`
- Remove _this default text_ from the PR description, and replace it with a summary of the changes (e.g. 'Added 10 new speakers, Added 6 sponsors, Added emojis to CFP page').

*If you are adding or removing organizers*, send an email to info@devopsdays.org with the full names, email addresses, and organizations the added/removed persons represent. (The global team uses this information to maintain your mailing list as well as the Slack.)

Up to date guidelines for pull requests can be found in the [contributing guide](https://github.com/devopsdays/devopsdays-web/blob/main/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: data/events/2024-nairobi.yml
Comment (somatorio) [inline data/events/2024-nairobi.yml:1]: This file moved to `data/events/2024/nairobi/main.yml`

**Comment by glasnt** [comment]
Hi! It looks like this PR is from an out of date fork. You'll need to [sync your fork](https://help.github.com/articles/syncing-a-fork/) and update your PR.

**Comment by phrawzty** [comment]
Hello @alvo254. This PR has been open for two weeks now. How can we help you make the necessary modifications in order to prepare it for merge? Please let us know how you'd like to proceed; otherwise I'll go ahead and close the PR for inactivity.

**Comment by phrawzty** [comment]
As this PR has been open for three weeks without any response, I'm going to go ahead and close it. Please feel free to reach out via email or Slack if you require further assistance. cc @adrianmoisey

**Pull request closed without merging**